### DISCUSSION TITLE
Problems compiling firmware with MESH_EDIT_GFX_OVERLAY

### DISCUSSION THREAD

**Discussion by derykyjw** [Jyers/Marlin#1325] [Q&A] [answered]
Hi, I'm trying to compile the latest version of the firmware myself with UBL enabled. However when I uncomment MESH_EDIT_GFX_OVERLAY, the code refuses to compile and throw the error below: 
#error "MESH_EDIT_GFX_OVERLAY requires AUTO_BED_LEVELING_UBL and a Graphical LCD."

I do have AUTO_BED_LEVELING_UBL uncommented and a LCD display so i’m not sure why it’s throwing the error

I have tried to comment MESH_EDIT_GFX_OVERLAY, and the code compiles fine. Any help would be appreciated. 

Here is the error that is showing up.

<img width="943" alt="Screen Shot 2021-08-20 at 5 58 13 pm" src="https://user-images.githubusercontent.com/39140824/130200464-dbaebdb9-4336-4d4f-a334-36d4eb4339ed.png">

**Comment by Cal-LF** [comment, marked as answer]
It is telling you the screen  is not compatible. A change to the E3V2 screen might work.

**Reply by derykyjw** [reply]
so does MESH_EDIT_GFX_OVERLAY not work with stock ender3v2 screen? is this the setting used to show the mesh map on the printer screen?
